src/dialogs/quick-bar/ha-quick-bar.ts

```diff
@@ -516,6 +585,27 @@ export class QuickBar extends LitElement {
           color: var(--primary-text-color);
         }
 
+        span.command-category {
+          font-weight: bold;
+          padding: 3px;
+          display: inline-flex;
+          border-radius: 6px;
+          color: black;
+        }
+
+        .command-category.reload {
+          --ha-chip-background-color: pink;
```

Comment:
We should probably make this one themeable too...
```suggestion
          --ha-chip-background-color: #cddc39;
```